addons/mail/static/src/owl/components/composer_text_input/composer_text_input.scss

```diff
@@ -15,17 +15,40 @@
     overflow: auto
 }
 
-.o_ComposerTextInput_mentionMenuItem {
-    white-space: nowrap;
-    overflow: hidden;
+.o_ComposerTextInput_mention_dropdown .o_mention_proposition {
 
-    &.o-mobile {
-        padding: 5px;
+    display: flex;
+    width: 100%;
+    padding: 3px 15px;
+    .o_mention_updatable_im_status {
+        padding-right: 3px;
     }
+    .o_mention_name {
+        font-weight: 500;
+        padding-right: 3px;
+    }
+    .o_mention_info {
+        font-style: italic;
+        color: gray;
```

Comment:
`gray('...')` pick one value that exists and is close visually

Reply:
Done